guacamole/src/main/frontend/src/app/client/types/ManagedClient.js

```diff
@@ -174,14 +175,25 @@ angular.module('client').factory('ManagedClient', ['$rootScope', '$injector',
          * @type ManagedFilesystem[]
          */
         this.filesystems = template.filesystems || [];
-        
+
         /**
-         * All messages that have been sent to the client that should be
-         * displayed.
-         * 
-         * @type ManagedClientMessage[]
+         * The current number of users sharing this connection, excluding the
+         * user that originally started the connection. Duplicate connections
+         * from the same user are included in this total.
+         */
+        this.userCount = template.userCount || 0;
+
+        /**
+         * All users currently sharing this connection, excluding the user that
+         * originally started the connection. If the connection is not shared,
+         * this object will be empty. This map consists of key/value pairs
+         * where each key is the user's username and each value is an object
```

Comment:
Too bad we can't just use `Set`. I guess it's still a little too new for some browsers (looking at you IE and Edge < 12)